packages/blade/src/components/Tour/Tour.stories.tsx

```diff
@@ -132,151 +315,740 @@ const TourTemplate = () => {
+            footer: ({ stopTour }) => {
+              return (
+                <Button
+                  size="small"
+                  onClick={() => {
+                    stopTour();
+                  }}
+                >
+                  Got it
+                </Button>
+              );
+            },
+          }
+        : {
+            name: 'start-tour',
+            title: 'Tour Complete!',
+            content: () => {
+              return (
+                <Text>
+                  You have completed the tour. You can find it here when you want to take it.
+                </Text>
+              );
+            },
+            footer: ({ stopTour }) => {
+              return (
+                <Button
+                  size="small"
+                  onClick={() => {
+                    stopTour();
+                  }}
+                >
+                  Thanks.
+                </Button>
+              );
+            },
+          },
+    ],
+    [isTourSkipped],
+  );
+
+  return (
+    <Box>
+      <Tour
+        steps={steps}
+        isOpen={isOpen}
+        activeStep={activeStep}
+        onFinish={() => {
+          console.log('finished');
+          setIsOpen(false);
+          setIsTourSkipped(false);
+          setActiveStep(0);
+        }}
+        onOpenChange={({ isOpen }) => {
+          console.log('open change', isOpen);
+          setIsOpen(isOpen);
+        }}
+        onStepChange={(step) => {
+          console.log('step change', step);
+          setActiveStep(step);
+        }}
+      >
+        <TourStep name="start-tour">
+          <Button
+            marginBottom="spacing.5"
+            onClick={() => {
+              setIsOpen((prev) => !prev);
+            }}
+          >
+            {isOpen ? 'Tour In Progress' : 'Start Tour'}
+          </Button>
+        </TourStep>
+        <Text>
+          You can create complex flows like interruptible tours by dynamically modifying the steps
+          array, and changing it's contents.
+        </Text>
+        <Text>
+          Compose and make use of methods provided by the tour component like{' '}
+          <Code size="medium">stopTour</Code>, <Code size="medium">goToStep</Code>,{' '}
+          <Code size="medium">goToNext</Code> etc to control the behaviour of the current tour step
+        </Text>
+        <Center>
+          <Box display="flex" gap="spacing.4" alignItems="stretch">
+            <TourStep name="step-1">
+              <Box padding="spacing.4" backgroundColor="surface.background.level2.lowContrast">
+                Step 1
+              </Box>
+            </TourStep>
+            <TourStep name="step-2">
+              <Box padding="spacing.4" backgroundColor="surface.background.level2.lowContrast">
+                Step 2
+              </Box>
+            </TourStep>
+          </Box>
+        </Center>
+      </Tour>
+    </Box>
+  );
+};
+InterruptibleTour.storyName = 'Product Usecase: Interruptible Tour';
```

Comment:
Just call it Interruptible tour
```suggestion
InterruptibleTour.storyName = 'Interruptible Tour';
```

Reply:
This is an existing product [usecase](https://github.com/razorpay/dashboard/blob/44a954660b0d851cd5fedf48b44d85731e5c48ea/web/js/merchantLA/containers/MerchantTour/index.js#L123-L138) demo. 

I like to prefix the "Product Usecase" to convey this is not a "prop" which can be switched on or off, but rather a whole usecase that consumers create by composing various blade components.